adapters/ratalert/index.js

```diff
@@ -0,0 +1,110 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Matic = require("../../sdk/matic");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+
+class RatAlert {
+    constructor() {
+        this.name = "ratalert";
+        this.symbol = "ETH";
+        this.token = "0x7ceb23fd6bc0add59e62ac25578270cff1b9f619"; // WETH
+        this.protocol = "polygon";
+        this.block = 28184455;
+        this.contract = "0x3242CCF5d7f35BaB5A560e7ae2Fa64EfDCa9Cd1c"; // RATCAST
```

Comment:
All addresses must be lowercased (provider address, NFT contract address, buyer address, ...)